### DISCUSSION TITLE
Trilium boot terribly slow after update [win]

### DISCUSSION THREAD

**Discussion by giuseppepagnoni** [TriliumNext/Trilium#3260] [Q&A]
HI, I've updated to the latest version 0.56.2 and now the boot time is above 30 secs... previous version 0.55.1 was just a few secs on my pc.
Any idea?
Thanks

**Comment by XXXJetfireXXX** [comment]
Was it the first boot after the update? After each update Trillium creates a before-migration backup file on the first boot to prevent any damage that could be caused to your database by a new version. The larger your database is, the longer it takes to make a backup.
![image](https://user-images.githubusercontent.com/43646178/198843316-8877c74a-3050-4990-bfb2-87943b7a2f25.png)

Try booting it up a few more times. If the problem still persists, then I suggest you to create a bug report with logs attached.

**Reply by zadam** [reply]
There was no migration in the 0.56 compared to 0.55 though. Not sure what it could be...

**Reply by XXXJetfireXXX** [reply]
Oh, my bad. I was thinking that it creates a new backup after each update x)
Then it indeed is something different...

**Comment by sigaloid** [comment]
Perhaps try launching it in a command line to see if there are any relevant errors or to see between which lines of output the delay is in.

**Comment by giuseppepagnoni** [comment]
Thanks for the answers. I launched from the command line and here's what I get:

![Snipaste_2022-10-30_09-16-56](https://user-images.githubusercontent.com/48069163/198869100-f0f36d3b-c633-4e10-b9bd-6b7915a0a84c.jpg)

Most of the time lost during the boot is after the line that begins with "generated" appears. The next line, the first beginning with 200, appears only after 15-20 secs.

I've reinstalled version 0.55.1 and boot time is fast as I remembered, just a few secs. So difference is from 0.55 to 0.56. I'm running on Win 10.

**Reply by sigaloid** [reply]
That sounds like not a backend issue - nothing much with the database is happening. Perhaps an electron / GUI problem. Were there any changes between versions related to the GUI - maybe a library was updated? @zadam

**Reply by zadam** [reply]
Not much has changed, electron is the same version in 0.55.1 and 0.56 ...

it is indeed strange, based on the log everything seems to be very fast.
